src/6_branch.js

```diff
@@ -834,7 +834,7 @@ Branch.prototype['logout'] = wrap(callback_params.CALLBACK_ERR, function(done) {
 		self.sessionLink = data['link'];
 		self.session_id = data['session_id'];
 		self.identity_id = data['identity_id'];
```

Comment:
Wouldn't we want to drop `identity_id` too for conciseness? If it's stored in local storage, if we leave `identity_id` is it possible for it to be attributed on pageviews in addition to web session start?

Reply:
I forget which one belongs to which. IIRC  identity_id -> v1/pageview, identity -> v1/open / web session start

Reply:
The Android SDK, like the iOS SDK, also stores the `identity_id` received from the server. https://github.com/BranchMetrics/android-branch-deep-linking-attribution/blob/master/Branch-SDK/src/main/java/io/branch/referral/ServerRequestLogout.java#L52

If memory serves we just added developer identity to /v1/open for the purpose of populating it in web session starts. That broke a regression, but we concluded it was the right thing to do and adjusted the regression. It was also added to pageviews for the same reason, though that didn't break a regression.

So that means that as long as `identity` is null, there will be no developer identity, even if `identity_id` is non-null. And finally, the session is retained through a logout/change of developer identity. Doesn't that get tied to an `identity_id`? I'm not sure.

Reply:
Though after all of that, it looks like the server isn't returning a developer identity, so even without this code change, it should be nulled out. I'll need to dig further.
```json
{"session_id":"804487127261343793","identity_id":"804487127236518275","link":"https://bnctestbed.app.link/?%24identity_id=804487127236518275"}
```

Reply:
Yeah, the `identity` is still being sent in pageviews and opens even with this change. More to come.

Reply:
LMK if I can help with testing.